src/pattern-matching/destructuring-arrays.md

```diff
@@ -5,3 +5,10 @@ You can destructure arrays, tuples, and slices by matching on their elements:
 ```rust,editable
 {{#include ../../third_party/rust-by-example/destructuring-arrays.rs}}
 ```
+
+<details>
+  
+Key points:
+* Continuing our journey in pattern matching, this slide just shows another type that can destructured with pattern matching. 
```

Comment:
Let's take this line out: it's more narrative than a "key point" for the instructor.